salt/modules/boto_dynamodb.py

```diff
@@ -233,7 +265,7 @@ def describe(table_name, region=None, key=None, keyid=None, profile=None):
     return table.describe()
 
 
-def _extract_index(index_data, global_index=False):
+def extract_index(index_data, global_index=False):
```

Comment:
That's a good point. I think that's a good way to approach it since this function is specific to only the dynamodb module and state.